apps/microfrontend-platform-devtools/src/app/find-capabilities/find-capabilities.component.scss

```diff
@@ -5,14 +5,11 @@
   > sci-viewport {
     flex: auto;
 
-    section {
+    &::part(content) {
```

Comment:
Just for my understanding: This has to be done this way because `sci-viewport` has a `part` with the name `content`, so this selector refers to the content projected into the `slot`.
```
<div #viewport_client class="viewport-client" part="content">
  <slot></slot>
</div>
```

It is not possible anymore to style the content the way it was done before, because it is contained a ShadowDOM, right?

Reply:
This change is not related to the issue. I made the change because I wanted to get rid of the section element that we added to lay out our content. The selector `sci-viewport::part(content)` provides a convenient way to style the parent of slotted content: `<div #viewport_client">`.